### DISCUSSION TITLE
Ackowledge groups of alerts but only for specific hosts

### DISCUSSION THREAD

**Discussion by launchdaemon** [Security-Onion-Solutions/securityonion#2879] [Unsupported Versions] [answered]
Hi,

I am quite new to Security Onion so hope this isn't obvious but have tried to search back through the discussions in case this has been mentioned. #2117 is related but not as specific as this.

I have a test server running 2.3.21 that was a network install on Ubuntu 18.04. so-status reports all services running.

In the Alerts window I can see alerts initially grouped by Name and Module.  This is fine and works when I want to acknowledge all instances of an alert regardless of host.  If I drill down in to a specific alert to see the different instances/ hosts I can see more detail which is great.  The problem I am having is that once I am drilled down in to an alert, if I then group by source IP for example the Acknowledge option is no longer there.

Is this expected?  I want to be able to acknowledge a large number of alerts that I know are ok for a specific host but keep the rest to investigate further.

I have tried in Firefox and Chrome to rule out a browser quirk.

Thanks

**Comment by jertel** [comment, marked as answer]
Yes, it's expected. Since you're interested in all alerts from a source IP you can change your Alert query to the "Group By Source IP, Name" selection. 

![image](https://user-images.githubusercontent.com/16639611/107247656-80001c80-69ff-11eb-97d7-63cc01769f9e.png)

You're now viewing a list of alerts grouped by source IP and rule name. So you're closer, but you want to dive into a specific host, so click on the IP of interest, and then choose Include on the popup menu. You'll now see the following:

![image](https://user-images.githubusercontent.com/16639611/107249158-154fe080-6a01-11eb-8efe-22d3e7480cc9.png)

Now you're looking at all the alerts for that one host. But suppose you have 100s of different alerts firing for that one host. You can drop the group by for rule.name by clicking the X in the red bubble containing "Group: rule.name" label.

![image](https://user-images.githubusercontent.com/16639611/107249347-49c39c80-6a01-11eb-8683-a1e8fdfda605.png)

Now you are left with all alerts for that one host, grouped by rule severity. That's as far as you can reduce it. You will now be able to acknowledge those three groups.

Let me know if this helps.

**Reply by launchdaemon** [reply]
Hi @jertel,
Thanks for your reply and apologies taking so long to respond.  I should have looked more at the pre-defined filtering options but your information was very helpful.

Can I ask why filtering or grouping by certain categories loses you the ability to acknowledge those alerts?

Thanks

**Reply by jertel** [reply]
It's not that certain categories lose the ability to acknowledge. The issue arises when you stop grouping by alert severity.
